npm-react-codemod/react-codemod

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+DIR=$(npm root -g)/react-codemod
+TRANSFORM=$1
+shift
+
+$DIR/node_modules/.bin/jscodeshift -t $DIR/build/$TRANSFORM.js $@
```

Comment:
I'm pretty sure this isn't going to work on Windows. I thought we were going to make this a js script that used node to shell out to jscodeshift?